net/tcp/tcp_recvwindow.c

```diff
@@ -224,14 +203,49 @@ uint16_t tcp_get_recvwindow(FAR struct net_driver_s *dev,
   return tcp_calc_rcvsize(conn, recvwndo);
 }
 
+/****************************************************************************
+ * Name: tcp_get_scalewindow
```

Comment:
i'm not sure if i like this name.
how about tcp_get_unscaled_recvwindow?

Reply:
lets' expose only one function here but not both:

1. tcp_get_recvwindow return the bytes(after UINT16_MAX process)
2. or tcp_get_scaled_recvwindow return the scaled value(after UINT16_MAX process too)

Reply:
Ok, rename the tcp_get_scalewindow to tcp_get_scaled_recvwindow